torchmetrics/metric.py

```diff
@@ -143,6 +143,9 @@ def add_state(
         elif dist_reduce_fx is not None and not isinstance(dist_reduce_fx, Callable):
             raise ValueError("`dist_reduce_fx` must be callable or one of ['mean', 'sum', 'cat', None]")
 
+        if isinstance(default, Tensor):
+            default = default.contiguous()
```

Comment:
it should already be converted before syncing between devices:
https://github.com/PyTorchLightning/metrics/blob/284e39ab7f9da6dc829e4d47311261fc74560d1b/torchmetrics/utilities/distributed.py#L108

Reply:
I just thought it would be slightly safer.

Reply:
fine by me :]